src/gtc/gtcpp/gtcpp.py

```diff
@@ -230,7 +235,7 @@ class IJCache(LocNode):
 
 class GTMultiStage(LocNode):
     loop_order: common.LoopOrder
-    stages: List[GTStage]  # TODO at least one
+    stages: List[GTStage]
```

Comment:
Removing the TODO is a conscious decision: if we require at least one, we need to do this in all testing code. I am not sure if we should. Currently we have tests, that ensure that generated code for the case of not stages (and the case of no multi-stages compiles.